core/src/VeyonConfigurationProperties.h

```diff
@@ -119,7 +119,8 @@
 	OP( VeyonConfiguration, VeyonCore::config(), bool, enforceSelectedModeForClients, setEnforceSelectedModeForClients, "EnforceSelectedModeForClients", "Master", false, Configuration::Property::Flag::Standard )	\
 	OP( VeyonConfiguration, VeyonCore::config(), bool, autoOpenComputerSelectPanel, setAutoOpenComputerSelectPanel, "AutoOpenComputerSelectPanel", "Master", false, Configuration::Property::Flag::Standard )	\
 	OP( VeyonConfiguration, VeyonCore::config(), bool, confirmUnsafeActions, setConfirmUnsafeActions, "ConfirmUnsafeActions", "Master", false, Configuration::Property::Flag::Standard )	\
-
+	OP( VeyonConfiguration, VeyonCore::config(), bool, openOnMasterScreen, setOpenOnMasterScreen, "openOnMasterScreen", "Master", false, Configuration::Property::Flag::Standard )	\
```

Comment:
The actual property name should start with a capital letter, i.e. `"OpenOnMasterScreen"`. Besides this I'd suggest to make it more descriptive and generic at the same time, e.g. `"ShowFeatureWindowsOnSameScreen"` (and of course corresponding function names).

Reply:
Done.